app/javascript/components/widgets/twitter/tweet_body.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { fontSizes } from '../../../lib/theme';
+
+const TweetText = styled.div`
+  color: #959292;
+  font-size: ${fontSizes.large};
+  margin-bottom: 29px;
+`;
+const getStatusLink = text => {
+  const allWords = text.split(/[\s,\n]+/);
+  return allWords.filter(word => word.includes('https://t.co/'))[0];
+};
+
+const TweetBody = ({ text }) => {
+  const displayText = text.replace(getStatusLink(text), ' ');
```

Comment:
You mentioned [1] that getStatusLink replacement is happening in the backend now but looks like its still used:
https://github.com/mojotech/helios2/pull/130/commits/e95b3bedaddd1d5059074dc55a3251f0084f5edc#r302201079

Reply:
Ah okay, I think it's snuck it's way into an older commit so I didn't see it. I'll remove it.